apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/DMEClaimTransformerTest.java

```diff
@@ -40,21 +72,12 @@ public void transformSampleARecord() throws FHIRException, IOException {
     List<Object> parsedRecords =
         ServerTestUtils.parseData(Arrays.asList(StaticRifResourceGroup.SAMPLE_A.getResources()));
     DMEClaim claim =
-        parsedRecords.stream()
-            .filter(r -> r instanceof DMEClaim)
-            .map(r -> (DMEClaim) r)
-            .findFirst()
-            .get();
-
-    ExplanationOfBenefit eob =
-        DMEClaimTransformer.transform(
-            new TransformerContext(
-                new MetricRegistry(),
-                Optional.of(true),
-                FdaDrugCodeDisplayLookup.createDrugCodeLookupForTesting(),
-                new NPIOrgLookup()),
-            claim);
-    assertMatches(claim, eob, Optional.of(true));
+        (DMEClaim)
+            parsedRecords.stream().filter(r -> r instanceof DMEClaim).map(r -> r).findFirst().get();
```

Comment:
Another `r -> r` mapping.  Also consider using `cast` as mentioned in another test.

Reply:
actually, my move over to using Intellij had some unwelcome and painful steps...this one was one of the MOST painful...upon saving, the cast inside the lambda would get nuked...to at least get things compile, I had to put that cast outside the lambda. This has got to be a function of the formatter, but if I am the only one experiencing this, then I might need any customizations and/or custom formatting tweaks you have so I can get back to what the code used to be (and what it should be). I am using the 'checkstyle idea' 5.71.1

Reply:
static cast took care of this.